Problem3/Problem3.js

```diff
@@ -0,0 +1,7 @@
+function checkForDuplicates(array){
+    for (var i = 0; i < array.length; i++){
+        if (array.indexOf(array[i]) !== array.lastIndexOf(array[i]))
```

Comment:
Am inteles. Am gasit asta seara o posibila rezolvare utilizand functii din Underscore.Js. As dori sa stiu daca se poate sa refac problema si pana cand. Mentionez ca maine(04.01.2015) voi fi plecat in delegatie in jud. Arges.
Multumesc.